src/objecttypes/setup_configuration/steps/sites.py

```diff
@@ -0,0 +1,60 @@
+import logging
+
+from django.contrib.sites.models import Site
+from django.core.exceptions import ValidationError
+from django.db import IntegrityError
+
+from django_setup_configuration.configuration import BaseConfigurationStep
+from django_setup_configuration.exceptions import ConfigurationRunFailed
+
+from objecttypes.setup_configuration.models.sites import SitesConfigurationModel
+
+logger = logging.getLogger(__name__)
+
+
+class SitesConfigurationStep(BaseConfigurationStep[SitesConfigurationModel]):
+    """
+    Configure the application site/domain.
+    """
+
+    namespace = "objecttypes_sites"
+    enable_setting = "objecttypes_sites_config_enable"
```

Comment:
```suggestion
    namespace = "sites_config"
    enable_setting = "sites_config_enable"
```
See https://github.com/maykinmedia/objects-api/pull/489#discussion_r1879866970